charrua-client-lwt.opam

```
@@ -1,9 +1,9 @@
 opam-version: "2.0"
 maintainer:   ["Mindy Preston"]
 authors   :   ["Mindy Preston"]
-homepage:     "https://github.com/mirage/charrua-core"
-bug-reports:  "https://github.com/mirage/charrua-core/issues"
-dev-repo:     "git+https://github.com/mirage/charrua-core.git"
+homepage:     "https://github.com/mirage/charrua"
+bug-reports:  "https://github.com/mirage/charrua/issues"
+dev-repo:     "git+https://github.com/mirage/charrua.git"
 tags:         [ "org:mirage"]
 doc:          "https://docs.mirage.io"
```

```suggestion
doc:          "https://mirage.github.io/charrua-core/"
```

and it will work with dune-release (need to make this across the other opam files too)